integration/airflow/tests/extractors/test_athena_extractor.py

```diff
@@ -0,0 +1,329 @@
+# Copyright 2018-2023 contributors to the OpenLineage project
+# SPDX-License-Identifier: Apache-2.0
+
+import json
+from unittest import mock
+
+import pytest
+from openlineage.airflow.extractors.athena_extractor import AthenaExtractor
+from openlineage.airflow.utils import is_airflow_version_enough
+from openlineage.client.facet import (
+    SymlinksDatasetFacet,
+    SymlinksDatasetFacetIdentifiers,
+)
+from openlineage.common.dataset import Dataset, Field, Source
+from openlineage.common.models import DbColumn, DbTableSchema
+from openlineage.common.sql import DbTableMeta
+
+from airflow import DAG
+
+if is_airflow_version_enough("2.2.4"):
```

Comment:
Any reason this is 2.2.4 here but skips on 2.2.1?

Reply:
Hi @JDarDagran do you mean 2.1.4 ? 

Since AthenaHook is not compatible with airflow 2.1.4 I want to run these tests if airflow version >= 2.2.4

Tests for airflow 2.2.4:
<img width="1173" alt="image" src="https://github.com/OpenLineage/OpenLineage/assets/130998593/23855119-7869-46ca-afd8-f66b8aa476e8">

Tests for airflow 2.1.4:
<img width="1147" alt="image" src="https://github.com/OpenLineage/OpenLineage/assets/130998593/5b6bad4b-c530-4a03-a03a-cf7b1f1f5ce9">

Reply:
Oh just a typo on my side, i'll fix this right now.

Thanks for noticing it 


```

Reply:
Thank you!